packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -120,28 +121,39 @@ export function applyPopupMenuStyles(): ICSSRules<{}> {
         background: colors.menuGray,
         zIndex: "1",
         top: toPx(55),
-        "& li > button": {
-            cursor: "pointer",
-            display: "block",
-            ...localizePadding(12, 12, 12, 40),
-            position: "relative",
-            border: "none",
-            background: "transparent",
-            textAlign: "left",
-            "&:before": {
-                position: "absolute",
-                content: "''",
-                opacity: ".6",
-                pointerEvents: "none",
-                top: toPx(12),
-                width: toPx(16),
-                height: toPx(16),
-                left: toPx(12),
-                /* tslint:disable-next-line */
-                background: "url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTMiIGhlaWdodD0iMTYiIHZpZXdCb3g9IjAgMCAxMyAxNiIgdmVyc2lvbj0iMS4xIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIj48ZyBpZD0iQ2FudmFzIiBmaWxsPSJub25lIj48ZyBpZD0iJiMyMzg7JiMxNTk7JiMxMzE7Ij48cGF0aCBkPSJNIDE0IDQuMjg5MDZMIDE0IDE2TCAxIDE2TCAxIDBMIDkuNzEwOTQgMEwgMTQgNC4yODkwNlpNIDEwIDRMIDEyLjI4OTEgNEwgMTAgMS43MTA5NEwgMTAgNFpNIDEzIDE1TCAxMyA1TCA5IDVMIDkgMUwgMiAxTCAyIDE1TCAxMyAxNVoiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC0xIDApIiBmaWxsPSJibGFjayIvPjwvZz48L2c+PC9zdmc+) center no-repeat"
+        "& li": {
+            "& > button": {
```

Comment:
if we target a class we do not need the direct child selector. Resolving in the dom becomes easier.

Reply:
Had a conversation with @nicholasrice about this, since these are shared functions used in several components. The only way to solve this would to be to break out all the individual elements into their own functions and then add all of those classes in the components that use them - which isn't necessarily better. Really, there should be a few more components (like pop-up menu) that gets the right styling. That is out of the scope of this task though. Leaving as is for now.